app/assets/favicons/browserconfig.xml

```diff
@@ -2,7 +2,7 @@
 <browserconfig>
     <msapplication>
         <tile>
-            <square150x150logo src="https://cdn.ons.gov.uk/sdc/v1.4.4/favicons/mstile-150x150.png"/>
+            <square150x150logo src="https://cdn.ons.gov.uk/sdc/v1.5.0/favicons/mstile-150x150.png"/>
```

Comment:
Does this need to be 1.5.1? Not sure if the other header/footer branch updated this file

Reply:
Yeah, you're right, basically anything with 1.5.0 should be 1.5.1 👍